CMakeLists.txt

```diff
@@ -190,6 +190,8 @@ find_package2(PUBLIC Protobuf)
 find_package2(PUBLIC msgpack)
 find_package2(PUBLIC Flatbuffers)
 
+find_package2(PUBLIC Vmc)
```

Comment:
Does this work? From the looks at the VMC project, it should be `find_package2(PUBLIC VMC)`?

Reply:
Fixed